src/Resource.php

```diff
@@ -507,17 +512,25 @@ protected function getToManyRelationship($name)
     // }}}
     // {{{ protected function initToOneRelationship()
 
-    protected function initToOneRelationship($name, $type)
-    {
+    protected function initToOneRelationship(
+        $name,
+        $type,
+        $auto_save = self::AUTO_SAVE_OFF
```

Comment:
Why are we defaulting to not saving? If a sub object is modified it will no longer be saved.